libraries/chain/db_notify.cpp

```diff
@@ -28,181 +28,232 @@ struct get_impacted_account_visitor
-   void operator()( const limit_order_create_operation& op ) {}
    void operator()( const limit_order_cancel_operation& op )
    {
-      _impacted.insert( op.fee_paying_account );
+      _impacted.insert( op.fee_payer() ); // fee_paying_account
+   }
+   void operator()( const call_order_update_operation& op )
+   {
+      _impacted.insert( op.fee_payer() ); // funding_account
    }
-   void operator()( const call_order_update_operation& op ) {}
    void operator()( const bid_collateral_operation& op )
    {
-      _impacted.insert( op.bidder );
+      _impacted.insert( op.fee_payer() ); // bidder
    }
    void operator()( const fill_order_operation& op )
    {
-      _impacted.insert( op.account_id );
+      _impacted.insert( op.fee_payer() ); // account_id
    }
    void operator()( const execute_bid_operation& op )
    {
-      _impacted.insert( op.bidder );
+      _impacted.insert( op.fee_payer() ); // bidder
    }
-
    void operator()( const account_create_operation& op )
    {
-      _impacted.insert( op.registrar );
+      _impacted.insert( op.fee_payer() ); // registrar
       _impacted.insert( op.referrer );
       add_authority_accounts( _impacted, op.owner );
       add_authority_accounts( _impacted, op.active );
    }
-
    void operator()( const account_update_operation& op )
    {
-      _impacted.insert( op.account );
+      _impacted.insert( op.fee_payer() ); // account
       if( op.owner )
          add_authority_accounts( _impacted, *(op.owner) );
       if( op.active )
          add_authority_accounts( _impacted, *(op.active) );
    }
-
    void operator()( const account_whitelist_operation& op )
    {
+      _impacted.insert( op.fee_payer() ); // authorizing_account
       _impacted.insert( op.account_to_list );
    }
-
-   void operator()( const account_upgrade_operation& op ) {}
+   void operator()( const account_upgrade_operation& op )
+   {
+      _impacted.insert( op.fee_payer() ); // account_to_upgrade
+   }
    void operator()( const account_transfer_operation& op )
    {
-      _impacted.insert( op.new_owner );
+      _impacted.insert( op.fee_payer() ); // account_id
+   }
+   void operator()( const asset_create_operation& op )
+   {
+      _impacted.insert( op.fee_payer() ); // issuer
    }
-
-   void operator()( const asset_create_operation& op ) {}
    void operator()( const asset_update_operation& op )
    {
+      _impacted.insert( op.fee_payer() ); // issuer
       if( op.new_issuer )
          _impacted.insert( *(op.new_issuer) );
    }
    void operator()( const asset_update_issuer_operation& op )
    {
-      _impacted.insert( op.new_issuer );
+      _impacted.insert( op.fee_payer() ); // issuer
+   }
+   void operator()( const asset_update_bitasset_operation& op )
+   {
+      _impacted.insert( op.fee_payer() ); // issuer
+   }
+   void operator()( const asset_update_feed_producers_operation& op )
+   {
+      _impacted.insert( op.fee_payer() ); // issuer
```

Comment:
Ideally producers should be added, but we can't simply do it, since it will break numbering of account history (like https://github.com/bitshares/bitshares-core/issues/265). Worth a new issue (label it `hardfork`).